python/step_3.html

```diff
@@ -0,0 +1,25 @@
+<div class="instructions">
+    <h2>Bottle</h2>
+    <p>
+        Bottle is a fast, simple and lightweight micro web-framework for Python. Now we are going to show you how to
+        define endpoints, receive data from a API call and call to external APIs.
+    </p>
+    <p>
+        Let's focus of few interesting concepts in bottle:
+    </p>
+    <ul>
+        <li>Routing: Requests to function-call mapping with support for clean and dynamic URLs.</li>
```

Comment:
these bullet points seems to me very difficult and scary terminology, can we be more informal?

Reply:
ahahah that was a copy-paste from their website